info.json

```diff
@@ -2,7 +2,7 @@
     "name": "knowledgeBase",
     "version": "1.0.1",
     "type": "solutionpack",
-    "local": false,
+    "local": true,
```

Comment:
What does this means?

Reply:
It determines if the solution pack is locally developed(true) or deployed from the marketplace(false).

Reply:
This however has no impact on solution pack execution, when SP is deployed via market place it automatically updates to false

Reply:
interesting...